tests/interactive_broker_version.py

```diff
@@ -42,6 +43,9 @@ def test_version(version, cmd=None, deploy=True, conf={}, debug=False,
 
     cluster = Cluster('LibrdkafkaTestCluster', root_path, debug=debug)
 
+    if args.conf.get('oauthbearer_method') == 'OIDC':
```

Comment:
There is no `args` in this method. Use `conf.get`  instead, and it should probably be `sasl_oauthbearer_method` to match the other code.